storage/src/journal/contiguous/fixed.rs

```diff
@@ -605,18 +605,6 @@ impl<E: Storage + Metrics, A: CodecFixed<Cfg = ()>> Journal<E, A> {
         Ok(())
     }
 
-    /// Syncs and closes all open sections.
-    pub async fn close(self) -> Result<(), Error> {
-        for (i, blob) in self.blobs.into_iter() {
```

Comment:
All non-tail blobs should be unmodified since their last write so I think this loop was not required to persist all journal data

Reply:
agree, in fact there's a longstanding open PR from andre to make these fully immutable!